stock_picking_comment_template_ext/models/stock_picking.py

```diff
@@ -1,15 +1,10 @@
 # Copyright 2020 Quartile Limited
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import api, fields, models
+from odoo import fields, models
 
 
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
-    note2 = fields.Html(compute="_compute_note2",)
-
-    @api.multi
-    def _compute_note2(self):
-        for picking in self:
-            picking.note2 = picking.group_id.sale_id.note2
+    note2 = fields.Html(default=lambda self: self.group_id.sale_id.note2)
```

Comment:
Does this lambda actually work?  If it does, then no need to extend _get_new_picking_values() in stock.move.

Reply:
I check again. 
Without both _prepare_procurement_values and _get_new_picking_values(), this did not work in this morning.

Reply:
> Without both _prepare_procurement_values and _get_new_picking_values(), this did not work in this morning.

That doesn't make sense to me. You should only need either.

Reply:
Oops, I meant "Without neither the two methods,  default=lambda did not work."

Reply:
now, We know only _get_new_picking_values() is needed and `default=`  is not required.